encoding/gocode/gen.go

```diff
@@ -289,7 +289,7 @@ func GenerateLineageBinding(lin thema.Lineage, cfg *BindingConfig) ([]byte, erro
 		CueModName:          cfg.CueModName,
 		LoadDir:             cfg.LoadDir,
 		EmbedPath:           cfg.EmbedPath,
-		CUEPath:             cfg.CuePath.String(),
+		CUEPath:             fmt.Sprintf("%q", cfg.CuePath.String()),
```

Comment:
oh well now i'm just _sure_ this never worked and i didn't test it properly 🤦🏻 

i think it's counterintuitive to expect any caller to have to add quotes on a string, though. Instead, let's add the quotes around where it's actually used in the template.

Reply:
Where else should I put that, then? Those are the variables used in the template, and with that change it works wether or not quotes were part of the lineage on the command line.

Reply:
wow, this is like my third time misreading something you did in the last couple days. i need to slow down 😭 

you're right, i'm wrong. i thought this was in `cmd/thema`. Here, this is fine - we're just constructing the private var to pass in to the template. This isn't exposed to external callers of `GenerateLineageBinding()`.